api/swagger.yaml

```diff
@@ -8876,13 +8876,19 @@ paths:
                   SubnetSize specifies the subnet size of the networks created from the default subnet pool
                 type: "integer"
                 format: "uint32"
+              DataPathPort:
+                description: |
+                  DataPathPort specifies the vxlan udp port number for vxlan traffic
```

Comment:
- s/vxlan/VXLAN/
- s/udp/UDP/
- Perhaps `data traffic` instead of `VXLAN traffic`


Also, can you

- describe the accepted port range (1024-...)
- describe that if no port is set, or is set to `0`, the default (4789) is used?